monitoring/mock_uss/scdsc/routes_injection.py

```diff
@@ -52,11 +60,10 @@ def query_operational_intents(
 
     updated_op_intents = []
     for op_intent_ref in get_details_for:
-        updated_op_intents.append(
-            scd_client.get_operational_intent_details(
-                resources.utm_client, op_intent_ref.uss_base_url, op_intent_ref.id
-            )
+        op_int, resp = scd_client.get_operational_intent_details(
```

Comment:
[get_operational_intent_details](https://github.com/interuss/dss/blob/03ec9e86979def793da35a9e07aadcad14435684/monitoring/monitorlib/clients/scd.py#L79) seems to return an OperationalIntent, not a tuple, so this change seems incorrect

Reply:
That's right. We made it a tuple, in our message_signing code. Wanted to keep that as a part of another PR. This line got overlooked while breaking the PR . Will need to fix.